lib/ClimaLandSimulations/src/utilities/inputs_dataframe.jl

```diff
@@ -1,11 +1,25 @@
 export make_inputs_df
 
 """
-    make_inputs_df(LOCAL_DATETIME, drivers)
+    make_inputs_df(site_ID)
 
 Returns DataFrames of inputs, one unitless, one in SI units, one in commonly used units. 
 """
-function make_inputs_df(LOCAL_DATETIME, drivers)
+function make_inputs_df(
+    site_ID;
+    FT = Float64,
+    site_setup_out = site_setup(site_ID),
```

Comment:
I noticed several functions where make these - should they be made once and passed? not sure, just asking! right now they are not expensive to make

Reply:
I understand it's a bit weird but I'll leave it like that for now - `make_inputs_df` is different because it uses unitful and creates input with SI or "commonly used" units, but I avoid this for `fluxnet_simulation()` where we might want efficiency (make_inputs_df is useful for e.g., plotting stuff). 
We could refactor later.